build/isc-bind9/build-911.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+#
+# Copyright 2017 OmniTI Computer Consulting, Inc.  All rights reserved.
+# Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/functions.sh
+
+PROG=bind
+VER=9.11.22
+PKG=network/bind-911
+SUMMARY="ISC BIND DNS Server & Tools"
+DESC="Server & Client Utilities for DNS"
+
+RUN_DEPENDS_IPS="
+    ooce/network/bind-common
+"
+
+set_arch 64
+
+SKIP_LICENCES="*"
+
+MAJVER=${VER%.*}            # M.m
+sMAJVER=${MAJVER//./}       # Mm
+PATCHDIR=patches-$sMAJVER
+
+OPREFIX=$PREFIX
+PREFIX+=/$PROG-$MAJVER
+
+HARDLINK_TARGETS="
+    opt/ooce/bind-9.11/bin/isc-config.sh
+    opt/ooce/bind-9.11/sbin/named
+    opt/ooce/bind-9.11/share/man/man1/isc-config.sh.1
```

Comment:
$PREFIX

Reply:
have used `${PREFIX#/}` to omit the beginning forward-slash from `$PREFIX`, otherwise build fails with errors whilst creating hardlinks